app/app-insights/log-trace-message.js

```diff
@@ -1,7 +1,5 @@
-const appInsights = require('applicationinsights')
```

Comment:
Can we rename the file `track-trace.js` to keep it consistent with the `track-exception.js`